src/test/java/core/basesyntax/service/impl/FruitTransactionServiceImplTest.java

```diff
@@ -0,0 +1,93 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitTransactionService;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+class FruitTransactionServiceImplTest {
+    private final FruitTransactionService fruitTransactionService =
+            new FruitTransactionServiceImpl();
+
+    @Test
+    void getFruitTransactionsFromFile_Ok() {
+        final List<FruitTransaction> actualList = fruitTransactionService
+                .getFruitTransactionsFromFile("src/test/resources/wrongData.txt");
+
+        final List<FruitTransaction> expectedList = new ArrayList<>();
+
+        FruitTransaction fruitTransaction1 = new FruitTransaction();
+        fruitTransaction1.setOperation(FruitTransaction.Operation.BALANCE);
+        fruitTransaction1.setFruit("banana");
+        fruitTransaction1.setQuantity(20);

```

Comment:
Your code would be much shorter if you had a constructor for fruitTransation

Reply:
I agree, but if I create a constructor of FruitTransaction class, then in the FruitTransactionServiceImpl class, the getFruitTransactionsFromFile() method will not be readable when I created the objects of FruitTransaction class, so I use setters there for better readability